NUnitLite/TouchRunner/TouchRunner.cs

```diff
@@ -140,14 +140,22 @@ public void AutoRun ()
 			});
 		}
 
+		bool running;
 		public void Run ()
 		{
+			if (running) {
+				Console.WriteLine ("Not running because another test run is already in progress.");
+				return;
+			}
+
 			if (!OpenWriter ("Run Everything"))
 				return;
 			try {
+				running = true;
```

Comment:
That should be earlier, before the `OpenWriter` call, otherwise the _problem_ won't be totally solved.

I'm not sure when/how does this happen, it that on bots ? otherwise it seems to be the UI that should be adjusted/disabled.